easybuild/easyconfigs/a/arpack-ng/arpack-ng-3.9.0-foss-2023b.eb

```diff
@@ -0,0 +1,37 @@
+# #
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+# #
+easyblock = 'ConfigureMake'
+
+name = 'arpack-ng'
+version = '3.9.0'
+
+homepage = 'https://github.com/opencollab/arpack-ng'
+description = "ARPACK is a collection of Fortran77 subroutines designed to solve large scale eigenvalue problems."
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = [GITHUB_SOURCE]
+sources = ['%(version)s.tar.gz']
+checksums = ['24f2a2b259992d3c797d80f626878aa8e2ed5009d549dad57854bbcfb95e1ed0']
+
+builddependencies = [
+    ('Autotools', '20220317'),
+    ('pkg-config', '0.29.2'),
+]
+dependencies = [
+    ('Eigen', '3.4.0'),
+]
+
+preconfigopts = "sh bootstrap && "
+configopts = '--enable-mpi --with-pic --with-blas="$LIBBLAS" --with-lapack="$LIBLAPACK"'
+
+github_account = 'opencollab'
```

Comment:
Please put this before the `source_urls`. It will work like this, since EasyBuild parses the full file before resolving the `GITHUB_SOURCE` template variable, but we have the convention of trying to stick to the order in which keywords are used as much as possible (it's easier for us humans :))